201-aci-linuxcontainer-volume-secret/azuredeploy.parameters.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "containergroupname": {
+      "value": "GEN-UNIQUE"
```

Comment:
for the param values here that are not globally unique, let's add some defaultValues in the template to simplify deployment